packages/speedometer.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Speedometer < Package
+  description 'Measure and display the rate of data across a network connection or data being stored in a file.'
+  homepage 'https://excess.org/speedometer/'
+  version '2.8'
```

Comment:
`version '2.8-3'`>

Reply:
the -3 is part of debian's versioning, the upstream package is just at 2.8